Jellyfin.Server/SocketSharp/RequestMono.cs

```diff
@@ -429,13 +439,13 @@ public override long Seek(long d, SeekOrigin origin)
                             real = position + d;
                             break;
                         default:
-                            throw new ArgumentException(nameof(origin));
+                            throw new ArgumentException("Unknown SeekOrigin value", nameof(origin));
                     }
 
                     long virt = real - offset;
                     if (virt < 0 || virt > Length)
                     {
-                        throw new ArgumentException();
+                        throw new ArgumentException("Invalid position", nameof(origin));
```

Comment:
I think bad position should be tied to `d`, not `origin` here:
```suggestion
                        throw new ArgumentException("Invalid position", nameof(d));
```